src/game/default/g_physics.h

```diff
@@ -26,6 +26,7 @@
 
 #ifdef __GAME_LOCAL_H__
 	#define DEFAULT_GRAVITY 800.0
+	void G_LiquidRipple(g_entity_t *ent, const vec3_t pos1, const vec3_t pos2, const vec_t size);
```

Comment:
I wanted `ent` optional, because some things like railgun beams *might* not work with that.